hoomd/mpcd/CollisionMethod.cc

```diff
@@ -48,15 +50,123 @@ void mpcd::CollisionMethod::collide(uint64_t timestep)
     // sync the embedded group
     m_cl->setEmbeddedGroup(m_embed_group);
 
+    // check for collision warnings
+    checkCollisionWarnings(timestep);
+
     // set random grid shift
     m_cl->drawGridShift(timestep);
 
     // update cell list
     m_cl->compute(timestep);
 
+    // create auxillary array for rigid bodies
+    beginRigidBodyCollision(timestep);
```

Comment:
You can wrap these functions here so you don't have to indent everything in them. It also will let us add an additional condition in the `if` statement that rigid bodies are actually present (we can worry about how to do this later)
```suggestion
    if (m_embed_group)
        {
        beginRigidBodyCollision(timestep);
        }
```